pages/glossary/modal-dialog.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Modal dialog
+key: modal-dialog
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+A modal dialog is a type of user interface element that appears on top of the main content of a [html web page][] or application and requires the user to take some action or make a decision before they can continue interacting with the program.
```

Comment:
```suggestion
A modal dialog is an element which, until it is closed, prevents user interaction with any content on the web page that is not a descendant element of the modal.  Visually, a modal often (but not necessarily) appears on top of all other content of the [html web page][] and visually darkens or otherwise obscures that other content.
```
Trying to put Carlos' advice into action.